eventhubs/client/azure-eventhubs/src/main/java/com/azure/eventhubs/ProxyAuthenticationType.java

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/**
+ * Supported methods of proxy authentication
+ */
+package com.azure.eventhubs;
+
+public enum ProxyAuthenticationType {
```

Comment:
The Javadoc should be above the class definition not the package name. I copied the documentation from my PR. I am sure Jonathan would have a comment about replacing the current descriptions for the enumeration.

```java
    /**
     * Supported methods of proxy authentication.
     */
    public enum ProxyAuthenticationType {
        /**
         * Proxy requires no authentication. Service calls will fail if proxy demands authentication.
         */
        NONE,
        /**
         * Authenticates against proxy with basic authentication scheme.
         */
        BASIC,
        /**
         * Authenticates against proxy with digest access authentication.
         */
        DIGEST,
    }
```

Reply:
Agreed. Please show more care to not make the documentation worse than it was previously. These mistakes are not acceptable - please take more care before submitting proposed changes to the code.